pkg/trace/stats/aggregation.go

```diff
@@ -25,6 +25,8 @@ const (
 type Aggregation struct {
 	BucketsAggregationKey
 	PayloadAggregationKey
+	// ExtraTagsKey is a comma-delimited string of all tag:value pairs for additional tags that should be used in stats aggregation.
+	ExtraTagsKey string
```

Comment:
Using a delimited string can break very easy if the delimiter is present in any of the tag values (which is entirely out of our control). Additionally, doing string manipulations for every aggregation could bring overhead of a bunch of extra allocations. Perhaps we should think about keying off a hash of the sorted tags?

Reply:
You're right, although we used to use such delimiters and strings in the old aggregation map long ago.

We could add the hash here and then add the `[]string` of custom tags to the `groupedStats` struct.